hudi-cli/src/main/scala/org/apache/hudi/cli/SparkHelpers.scala

```diff
@@ -56,8 +59,12 @@ object SparkHelpers {
   }
 
   def getBloomFilter(file: String, conf: Configuration): String = {
-    val footer = ParquetFileReader.readFooter(conf, new Path(file));
-    return footer.getFileMetaData().getKeyValueMetaData().get(HoodieAvroWriteSupport.HOODIE_AVRO_BLOOM_FILTER_METADATA_KEY)
+    val footer = ParquetFileReader.readFooter(conf, new Path(file))
+    val metadataMap: util.Map[String, String] = footer.getFileMetaData().getKeyValueMetaData()
```

Comment:
can we wrap this in a method in `ParquetUtils` class (where we already have methods for reading the keys etc)